src/Value/Color.php

```diff
@@ -256,4 +250,23 @@ private function allComponentsAreNumbers(): bool
 
         return true;
     }
+
+    /**
+     * Note that this method assumes the following:
+     * - The {@see aComponents} array has keys for `r`, `g` and `b`;
+     * - The values in the array are all instances of {@see Size}.
+     *
+     * Errors will be triggered or thrown if this is not the case.
+     */
```

Comment:
We can make the return type even more specific:

`@return non-empty-string`

Reply:
Done